pkg/cmd/provisioning/gcp/gcp.go

```diff
@@ -0,0 +1,31 @@
+package gcp
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+)
+
+type options struct {
+	TargetDir      string
+	PublicKeyPath  string
+	Region         string
+	Name           string
+	CredRequestDir string
+	DryRun         bool
+	Project        string
+}
+
+// NewGCPCmd implements the "gcloud" subcommand for the credentials provisioning
+func NewGCPCmd() *cobra.Command {
+	createCmd := &cobra.Command{
+		Use:   "gcloud",
```

Comment:
I don't have a strong opinion here, but I expected it to be `gcp` not `gcloud`. What do others think?

Reply:
+1 for gcp

Reply:
gcp it is